### ISSUE TITLE
Typos, missed imports, and broken lines

### ISSUE THREAD

**Issue by Gaaaavin** [NetoPedro/FocusFace#4]
In `model.py`, there are many typos and broken lines. Some imports are missed. e.g.

Line 14 should be 
`def forward(self, x, label, inference=False):`

Missed imports are
```
import torch
import torch.functional as F
from torch.nn.parameter import Parameter
```

Without these edits, the released code is impossible to reproduce

**Comment by NetoPedro** [comment]
Hi @Gaaaavin I will try to revise what the problem might be. It is most likely linked with the code cleaning process. If you have suggestions, feel free to submit a pull request too.

**Comment by Gaaaavin** [comment]
Hi @NetoPedro Thank you for the reply, I've also found many other issues digging deeper into the code. I don't think any pr can solve this. Hope you will update the code base soon. Thank you.
